### ISSUE TITLE
Fix codecov

### ISSUE THREAD

**Issue by richfitz** [mrc-ide/odin#97]
No longer working on travis:

```
Rscript -e 'covr::codecov()'
Error: identical(names(y), names) is not TRUE
Execution halted
```

**Comment by richfitz** [comment]
This can be triggered locally by installing a newer version of covr (the current CRAN version - 3.0.1).  I see the same error here with:

```
> cov <- covr::package_coverage()
Error: identical(names(y), names) is not TRUE
> traceback()
4: stop(sprintf(ngettext(length(r), "%s is not TRUE", "%s are not all TRUE"), 
       ch), call. = FALSE, domain = NA)
3: stopifnot(identical(names(y), names))
2: merge_coverage(trace_files)
1: covr::package_coverage()
```

**Comment by richfitz** [comment]
This will be fixed when #295 is on CRAN so using development covr might work

**Comment by richfitz** [comment]
```
tmp <- tempfile()
dir.create(tmp)
.libPaths()
.libPaths(c(tmp, .libPaths()))
.libPaths()
install.packages("covr")
cov <- covr::package_coverage()
```

vs

```
tmp <- tempfile()
dir.create(tmp)
.libPaths()
.libPaths(c(tmp, .libPaths()))
.libPaths()
devtools::install_github("r-lib/covr", upgrade = FALSE)
cov <- covr::package_coverage()
```
